SignalMessaging/categories/UIColor+OWS.m

```diff
@@ -142,72 +142,54 @@ + (UIColor *)ows_redColor
     return [UIColor colorWithRGBHex:0xf44336];
 }
 
+#pragma mark - GreyScale
+
 + (UIColor *)ows_whiteColor
 {
     return [UIColor colorWithRGBHex:0xFFFFFF];
 }
 
-+ (UIColor *)ows_light02Color
++ (UIColor *)ows_gray02Color
 {
-    return [UIColor colorWithRGBHex:0xF9FAFA];
+    return [UIColor colorWithRGBHex:0xF8F9F9];
 }
 
-+ (UIColor *)ows_light10Color
++ (UIColor *)ows_gray05Color
 {
     return [UIColor colorWithRGBHex:0xEEEFEF];
 }
 
-+ (UIColor *)ows_light35Color
-{
-    return [UIColor colorWithRGBHex:0xA4A6A9];
-}
-
-+ (UIColor *)ows_light45Color
-{
-    return [UIColor colorWithRGBHex:0x8B8E91];
-}
-
-+ (UIColor *)ows_light60Color
-{
-    return [UIColor colorWithRGBHex:0x62656A];
-}
-
-+ (UIColor *)ows_light90Color
++ (UIColor *)ows_gray15Color
```

Comment:
I don't see this one in the spec

Reply:
Ah yes, we just eliminated that one after Myles reviewed this work.